tasks/populate-db.mjs

```diff
@@ -23,17 +23,21 @@ cssdb.forEach(feature => {
 		feature.browser_support[browser] = browser_support[browser];
 	});
 
+	feature.browser_support = Object.assign({}, feature.browser_support, feature.browser_support_overrides);
```

Comment:
This is an escape hatch for incorrect MDN or caniuse data.

Ideally we would be patient and fix everything upstream, but then we will never be able to merge larger updates like this one.

Having an escape hatch makes it possible to unblock cssdb while we figure out what is going on upstream.

Reply:
I've made this more maintainable by adding `from` and `to` fields in `cssdb.settings.json`.

An override is now very specific and will only be applied if MDN has a specific value that we know and want to replace.

If the value doesn't match an exception is thrown.
This makes it possible to keep the amount of overrides small over time and clean these up each time MDN updates it's data.

Error message looks like this : `The overrides for android in any-link-pseudo-class might no longer be needed. MDN used to have ≤37 and now has 37.`

Reply:
This is great! Very well thought!